claim/gql_mutations.py

```diff
@@ -742,6 +752,7 @@ def async_mutate(cls, user, **data):
             raise PermissionDenied(_("unauthorized"))
         return set_claims_status(data['uuids'], 'review_status', 16)
```

Comment:
Can we use constants to represent the status/enums (replace 16 with more significant meaning)?

Reply:
Right, I replaced it with constants elsewhere in this PR. I'll update.

Reply:
This is actually on a different mutation. I'd rather not have too many unrelated changes in a hotfix. I'll update it in the LGTM branch for all values.
I even considered switching some of these definitions to Enums that are now properly supported to Graphene in Django 3 but I'm not sure that it won't have an impact on the existing API.